packages/block-editor/src/hooks/background.js

```diff
@@ -96,19 +97,38 @@ export function resetBackgroundImage( { attributes = {}, setAttributes } ) {
 	} );
 }
 
-function InspectorImagePreview( { label, url: imgUrl } ) {
+function InspectorImagePreview( { label, filename, url: imgUrl } ) {
 	const imgLabel = label || getFilename( imgUrl );
```

Comment:
`imgLabel` is now always `Background image`.

Reply:
Yea, we can consolidate that.